WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -476,7 +476,12 @@ public void onClick(View view) {
                         break;
                     default:
                         if (mOnPostButtonClickListener != null) {
-                            mOnPostButtonClickListener.onPostButtonClicked(buttonType, post);
+                            PostModel latestPost = mPostStore.getPostByLocalPostId(post.getId());
+                            if (latestPost == null) {
+                                loadPosts(LoadMode.FORCED);
+                                return;
+                            }
+                            mOnPostButtonClickListener.onPostButtonClicked(buttonType, latestPost);
```

Comment:
I'm not sure if this applies, but should we do the same [with Pages](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/fluxc-media-upload-service-post-list-fix/WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java#L363)?